[Exploration] Components colors and design updates

dalthviz opened the pull request:
- [x] issues: potencially fixes #14445 
- [ ] tests added / passed

### A preview:

|Widget|Before|After|
|-|-|-|
|Button|![bokeh_buttons_before](https://github.com/user-attachments/assets/c57763b0-a98a-4df7-a90e-1c61367792ef)|![bokeh_buttons_after](https://github.com/user-attachments/assets/d5637326-7683-4cef-97b0-89f2260c4d3b)|
|CheckboxButtonGroup|![bokeh_checkboxbuttons_before](https://github.com/user-attachments/assets/802c8561-ac89-4ed2-8b1a-320144f9aa80)|![bokeh_checkboxbuttons_after](https://github.com/user-attachments/assets/4da29c06-1093-4373-b593-0d4ee263fc94)|
|RadioButtonGroup|![bokeh_radiobuttons_before](https://github.com/user-attachments/assets/fb2a127f-5926-44fc-9e29-8a577a6f3e75)|![bokeh_radiobuttons_after](https://github.com/user-attachments/assets/0df816f4-092f-4c76-a777-d897f00b957d)|
|Toggle|![bokeh_toggle_before](https://github.com/user-attachments/assets/1e93722d-4893-4327-be26-7ab5fec5b008)|![bokeh_toggle_after](https://github.com/user-attachments/assets/5f388311-8b2a-437f-80e3-ed3fc798f61a)|
|Plot tools|![bokeh_tool_buttons_before](https://github.com/user-attachments/assets/7ee5f46d-110f-4e78-b514-84de55104104)|![bokeh_tool_buttons_after](https://github.com/user-attachments/assets/11aae1a7-0bb1-44ff-bb02-6707bc160616)|
|HelpButton|<img width="189" height="433" alt="imagen" src="https://github.com/user-attachments/assets/f2d207a2-308d-4430-a400-ebd8da49819f" />|<img width="186" height="430" alt="imagen" src="https://github.com/user-attachments/assets/e347b66a-375e-4421-a870-ceec3341e767" />|
|CheckboxGroup|![bokeh_checkboxgroup_before](https://github.com/user-attachments/assets/6ac054d8-b772-40a7-85d1-20ecee074a15)|![bokeh_checkboxgroup_after](https://github.com/user-attachments/assets/c9b1bfbf-c031-4509-8adc-59c8db28a434)|
|RadioGroup|![bokeh_radiogroup_before](https://github.com/user-attachments/assets/a01a69ea-14f5-4261-8e03-3fd4ccd6725b)|![bokeh_radiogroup_after](https://github.com/user-attachments/assets/80109457-f7cb-45c0-9c67-68175c72ce04)|
|Dropdown/Select*|![bokeh_dropdownselect_before](https://github.com/user-attachments/assets/d6438cf4-bf8b-4da8-9691-410f92438070)|![bokeh_dropdownselect_after](https://github.com/user-attachments/assets/bf0cccee-3c03-4a29-8034-8759329164bb)|
|Input*|![bokeh_input_before](https://github.com/user-attachments/assets/1de229fc-537e-4d84-8bf8-3fd76d29edc4)|![bokeh_input_after](https://github.com/user-attachments/assets/9d23f80c-7ba9-4d1b-bc33-4b81130b8488)|
|Tooltip|<img width="612" height="272" alt="imagen" src="https://github.com/user-attachments/assets/5dd1c604-6908-4969-8ac2-57b3b8caa1fd" />|<img width="608" height="192" alt="imagen" src="https://github.com/user-attachments/assets/38172f1a-9abc-49c2-a291-069a310c8a9f" />|
|Sliders*|![sliders_before](https://github.com/user-attachments/assets/7164beab-878e-42e0-b37e-a919942f76a8)|![sliders_after](https://github.com/user-attachments/assets/baa2dedb-792e-4ae9-991e-20896ca75f37)|
|Switch*|![switcher_before](https://github.com/user-attachments/assets/acb3cd6a-523a-4a9f-af22-0eb57dba9e64)|![switcher_after](https://github.com/user-attachments/assets/ef3cc19c-f40a-4c45-beed-c63baf3bef2b)|
|Tabs|![tabs_before](https://github.com/user-attachments/assets/f22fde19-ce31-4b94-addc-8acbd4bb6f6f)|![tabs_after](https://github.com/user-attachments/assets/da5d1f2a-d06e-4f42-a2d3-80b9eb67fce3)|
|Progress*|![progress_before](https://github.com/user-attachments/assets/b575652d-8178-48f3-8d6c-aec0a56f0a32)|![progress_after](https://github.com/user-attachments/assets/ef8d6c22-de9a-4183-b4b5-754e4818c04e)|

This is meant to follow the designs done at https://www.figma.com/design/3xk3QObj3XaALcHHyz8nmE/%F0%9F%93%8A-Bokeh-Design-System?node-id=482-368&p=f&t=HUYZW5Gqosl9Jsy7-0:
* General buttons: https://www.figma.com/design/3xk3QObj3XaALcHHyz8nmE/%F0%9F%93%8A-Bokeh-Design-System?node-id=278-3156&p=f&t=CLspZZjU3Rl9WUSR-0 
* Plot tools button: https://www.figma.com/design/3xk3QObj3XaALcHHyz8nmE/%F0%9F%93%8A-Bokeh-Design-System?node-id=1235-2736&t=HBRKZFSdRj2f7jJG-0
* Checkbox: https://www.figma.com/design/3xk3QObj3XaALcHHyz8nmE/%F0%9F%93%8A-Bokeh-Design-System?node-id=342-162&p=f&t=FVvCJGdGwKnBGNek-0
* Radio: https://www.figma.com/design/3xk3QObj3XaALcHHyz8nmE/%F0%9F%93%8A-Bokeh-Design-System?node-id=715-652&p=f&t=FVvCJGdGwKnBGNek-0
* Dropdown: https://www.figma.com/design/3xk3QObj3XaALcHHyz8nmE/%F0%9F%93%8A-Bokeh-Design-System?node-id=301-3618&t=0yl5fYz0yTGogEcD-0
* Input: https://www.figma.com/design/3xk3QObj3XaALcHHyz8nmE/%F0%9F%93%8A-Bokeh-Design-System?node-id=425-246&p=f&t=0yl5fYz0yTGogEcD-0
* Tooltip: https://www.figma.com/design/3xk3QObj3XaALcHHyz8nmE/%F0%9F%93%8A-Bokeh-Design-System?node-id=336-321&p=f&t=DRRoI25858288bB2-0
* Sliders: https://www.figma.com/design/3xk3QObj3XaALcHHyz8nmE/%F0%9F%93%8A-Bokeh-Design-System?node-id=425-163&t=HUYZW5Gqosl9Jsy7-0
* Switch: https://www.figma.com/design/3xk3QObj3XaALcHHyz8nmE/%F0%9F%93%8A-Bokeh-Design-System?node-id=425-289&t=InZfeSPgjTRqyOXC-0
* Tabs: https://www.figma.com/design/3xk3QObj3XaALcHHyz8nmE/%F0%9F%93%8A-Bokeh-Design-System?node-id=425-331&t=f7858O81yHdYd3Zc-0
* Progress: https://www.figma.com/design/3xk3QObj3XaALcHHyz8nmE/%F0%9F%93%8A-Bokeh-Design-System?node-id=425-204&p=f&t=naUJ1bb00AD1Xe3m-0

### Notes:

* Dropdown, select, general inputs, sliders, switch and progress partially customized. Further changes to those widgets/components would require more work not only over the styles side of things but the models code itself (addition of new elements or changing the position of current ones available like labels or descriptions for example).
* Customization of `select` style limited - https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/select#styling_with_css (possibility to use `base-select` but feature is experimental/not supported in the mayority of browsers)
* Customization of the sliders style limited since those are based on the classes defined in the noUiSlider library (maybe it could be possible to add more classes programatically to do further styling like changing handles for ranged vs single sliders)

dalthviz committed:
Buttons colors and focus ring updates

dalthviz committed:
CheckboxButtonGroup colors and focus ring updates

dalthviz committed:
Plot toolbar/tool buttons colors updates

dalthviz committed:
Update baselines

mattpap commented:
We've been discussing this internally with @philippjfr and @hoxbro, and our conclusion is that it would be beneficial to apply this new theme as an actual theme and not changing the defaults, at least for now. This way it will be easier to provide the new theme to the rest of the team and our users, without us having to commit to it right now, and there are things that we don't like about it. We also don't have a CSS based theming approach altogether and adding support for that, is a significant part of the grant, so it would be worthwhile to purse this sooner than later.

Let's think about how to approach this and we will discuss this during our technical meeting on Wednesday. In the meantime you can continue applying the theme to inputs, etc.

dalthviz committed:
Revert some color changes and add specific colors for plot tools

dalthviz committed:
Refine button group style following design clarifications

dalthviz committed:
Update baselines

dalthviz commented:
Although this is probably more like a exploratory PR than something to get merged now, leaving it as ready for review

As a side note, regarding approaches for a CSS based theming, started checking docs/examples and maybe https://docs.bokeh.org/en/latest/docs/user_guide/styling/dom.html#styling-dom-elements and https://github.com/bokeh/bokeh/pull/12907 could be related with that. Maybe a look to Panel templates could be useful too: https://panel.holoviz.org/explanation/styling/templates_overview.html

dalthviz commented:
Note: Now using this PR just as a way to preview how the design system done actually looks applied to Bokeh

dalthviz committed:
HelpButton color updates and handle icon color per button type

dalthviz committed:
Checkbox/Radio[Group] color and design updates

dalthviz committed:
Update baselines

dalthviz committed:
Revert input border-color change

dalthviz committed:
Update baselines

dalthviz committed:
Dropdown design updates. Changes possible for Select following dropdown design. Initial inputs general change (border on focus)

dalthviz committed:
Input colors update

dalthviz committed:
Fix validations

dalthviz committed:
Fix dropdown active/focus background-color

dalthviz committed:
Update baselines

dalthviz committed:
Keep border visible for inputs with prefix/suffix when disabled

dalthviz committed:
Tooltip colors updates

dalthviz committed:
Update baselines

dalthviz committed:
Initial styling for sliders

mattpap commented on bokehjs/src/less/tooltips.less:
```diff
@@ -106,28 +107,28 @@
 :host(.bk-right) .bk-arrow-inner {
   left: calc(-1 * var(--tooltip-arrow-border));
-  transform: translate(-100%, -50%);
+  transform: translate(-106%, -50%);
```
Where this 6% is coming from? To note, this transform is only meant to shift the origin for positioning.

dalthviz replied:
This was due to a gap between the arrow and the tooltip text element but will check then a way to prevent that without modifiying this :+1:

philippjfr replied:
>This was due to a gap between the arrow and the tooltip text element but will check then a way to prevent that without modifiying this 👍

That gap has annoyed me for so long, would love it if that could be fixed.

dalthviz replied:
Commit https://github.com/bokeh/bokeh/pull/14714/commits/64c5758db76d728bdb0a613222973f41e539c796 undos the change to the translate values and should take care of the gap

dalthviz committed:
Use calc instead of fixed values for sliders styling

dalthviz committed:
Undo tooltip arrow transform change. Prevent gap visibility with border

dalthviz committed:
Update baselines

dalthviz committed:
Switch style updates

dalthviz committed:
Merge branch 'branch-3.9' into dalthviz/14445_buttons_tools_design

dalthviz committed:
Update baselines

dalthviz committed:
Tabs style updates

dalthviz committed:
Merge branch 'branch-3.9' into dalthviz/14445_buttons_tools_design

dalthviz committed:
Update baselines

dalthviz committed:
Update tabs tests

dalthviz committed:
Update baselines

dalthviz committed:
Merge branch 'branch-3.9' into dalthviz/14445_buttons_tools_design

dalthviz committed:
Update limit value for codebase test

dalthviz committed:
Initial progress style updates

dalthviz committed:
Merge branch 'branch-3.9' into dalthviz/14445_buttons_tools_design

dalthviz committed:
Update baselines

dalthviz committed:
Update baselines

dalthviz committed:
Explore dark colors variant

dalthviz commented:
Discussing with Pavithra and Mateusz next steps for this PR, started exploring how a dark variant for the colors could be achieved/handled (and if more changes to the widgets rules are required). Giving that a check, the `light-dark` CSS function could be useful (https://developer.mozilla.org/en-US/docs/Web/CSS/Reference/Values/color_value/light-dark). Latest commit (https://github.com/bokeh/bokeh/pull/14714/commits/fc61eb0a20d059f88dc8ade05f8f24054d9251a9) explores using the `light-dark` function in the color definitions + the `color-scheme` rule and adds some changes to the widgets style rules (missing `background-color` `color` rules). With those changes, running something like:

<details>

```python
import numpy as np

from bokeh.plotting import figure, show

N = 4000
x = np.random.random(size=N) * 100
y = np.random.random(size=N) * 100
radii = np.random.random(size=N) * 1.5
colors = np.array([(r, g, 150) for r, g in zip(50 + 2 * x, 30 + 2 * y)], dtype=np.uint8)

TOOLS = "hover,crosshair,pan,wheel_zoom,zoom_in,zoom_out,box_zoom,undo,redo,reset,tap,save,box_select,poly_select,lasso_select,examine,fullscreen,help"

p = figure(width=200, height=200, tools=TOOLS, title="Title")
x = [1, 2, 3, 4, 5]
y1 = [2, 3, 4, 5, 6]
y2 = [3, 4, 5, 6, 7]

p.line(x, y1, legend_label="Temp.", line_color="blue", visible=False)
p.line(x, y2, legend_label="Objects", line_color="red")

p.legend.location = "top_left"
p.legend.border_line_color = "black"
p.legend.border_line_dash = "dotdash"
p.legend.border_line_width = 1
p.legend.item_background_policy = "odd"
p.legend.item_background_hatch_pattern = "dot"
p.legend.item_background_hatch_color = "green"
p.legend.item_background_hatch_scale = 5
# p.legend.item_background_fill_color = None

p.legend.click_policy = "hide"
p.legend.inactive_hatch_pattern = "vertical_line"
p.legend.inactive_hatch_scale = 5
# p.legend.inactive_fill_color = None

p.legend.background_hatch_pattern = "dot"
p.legend.background_hatch_color = "yellow"
# p.legend.background_fill_color = "red"

from bokeh.layouts import column, row
from bokeh.models import (
    BuiltinIcon,
    Button,
    CheckboxButtonGroup,
    Dropdown,
    HelpButton,
    RadioButtonGroup,
    Toggle,
    Tooltip,
)
from bokeh.models.dom import HTML

menu = [("Item 1", "item_1"), ("Item 2", "item_2"), None, ("Item 3", "item_3")]

checkbox_options = ["open", "high", "low", "close"]
radio_options = ["Option A", "Option B", "Option C"]

button = Button(label="default", button_type="default", icon=BuiltinIcon(icon_name="x"))
button_disabled = Button(label="default", button_type="default", disabled=True)
dropdown = Dropdown(label="default", button_type="default", menu=menu)
dropdown_disabled = Dropdown(label="default", button_type="default", menu=menu, disabled=True)
dropdown1 = Dropdown(label="primary", button_type="primary", menu=menu)
dropdown1_disabled = Dropdown(label="primary", button_type="primary", menu=menu, disabled=True)
dropdown2 = Dropdown(label="success", button_type="success", menu=menu)
dropdown2_disabled = Dropdown(label="success", button_type="success", menu=menu, disabled=True)
dropdown_split = Dropdown(label="default", button_type="default", menu=menu, split=True)
dropdown_split_disabled = Dropdown(
    label="default",
    button_type="default",
    menu=menu,
    split=True,
    disabled=True,
)
checkbox_grp = CheckboxButtonGroup(labels=checkbox_options, active=[0], button_type="default")
checkbox_grp_disabled = CheckboxButtonGroup(labels=checkbox_options, active=[0], button_type="default", disabled=True)
radio_button_group = RadioButtonGroup(labels=radio_options, active=0, button_type="default")
radio_button_group_disabled = RadioButtonGroup(labels=radio_options, active=0, button_type="default", disabled=True)
toggle = Toggle(label="default", button_type="default")
toggle_disabled = Toggle(label="default", button_type="default", disabled=True)

help_button = HelpButton(
    button_type="default",
    tooltip=Tooltip(
        content=HTML(
            """
This is a tooltip with additional information.<br />
It can use <b>HTML tags</b> like <a href="https://bokeh.org">links</a>!
""",
        ),
        position="right",
    ),
)
help_button_disabled = HelpButton(
    button_type="default",
    tooltip=Tooltip(
        content=HTML(
            """
This is a tooltip with additional information.<br />
It can use <b>HTML tags</b> like <a href="https://bokeh.org">links</a>!
""",
        ),
        position="right",
    ),
    disabled=True,
)
help_button1 = HelpButton(
    button_type="primary",
    tooltip=Tooltip(
        content=HTML(
            """
This is a tooltip with additional information.<br />
It can use <b>HTML tags</b> like <a href="https://bokeh.org">links</a>!
""",
        ),
        position="right",
    ),
)
help_button1_disabled = HelpButton(
    button_type="primary",
    tooltip=Tooltip(
        content=HTML(
            """
This is a tooltip with additional information.<br />
It can use <b>HTML tags</b> like <a href="https://bokeh.org">links</a>!
""",
        ),
        position="right",
    ),
    disabled=True,
)
help_button2 = HelpButton(
    button_type="success",
    tooltip=Tooltip(
        content=HTML(
            """
This is a tooltip with additional information.<br />
It can use <b>HTML tags</b> like <a href="https://bokeh.org">links</a>!
""",
        ),
        position="right",
    ),
)
help_button2_disabled = HelpButton(
    button_type="success",
    tooltip=Tooltip(
        content=HTML(
            """
This is a tooltip with additional information.<br />
It can use <b>HTML tags</b> like <a href="https://bokeh.org">links</a>!
""",
        ),
        position="right",
    ),
    disabled=True,
)

button1 = Button(label="primary", button_type="primary")
button1_disabled = Button(label="primary", button_type="primary", disabled=True)
checkbox_grp1 = CheckboxButtonGroup(labels=checkbox_options, active=[0], button_type="primary")
checkbox_grp1_disabled = CheckboxButtonGroup(labels=checkbox_options, active=[0], button_type="primary", disabled=True)
radio_button_group1 = RadioButtonGroup(labels=radio_options, active=0, button_type="primary")
radio_button_group1_disabled = RadioButtonGroup(labels=radio_options, active=0, button_type="primary", disabled=True)
toggle1 = Toggle(label="primary", button_type="primary")
toggle1_disabled = Toggle(label="primary", button_type="primary", disabled=True)

button2 = Button(label="success", button_type="success")
button2_disabled = Button(label="success", button_type="success", disabled=True)
checkbox_grp2 = CheckboxButtonGroup(labels=checkbox_options, active=[0], button_type="success")
checkbox_grp2_disabled = CheckboxButtonGroup(labels=checkbox_options, active=[0], button_type="success", disabled=True)
radio_button_group2 = RadioButtonGroup(labels=radio_options, active=0, button_type="success")
radio_button_group2_disabled = RadioButtonGroup(labels=radio_options, active=0, button_type="success", disabled=True)
toggle2 = Toggle(label="success", button_type="success")
toggle2_disabled = Toggle(label="success", button_type="success", disabled=True)

button3 = Button(label="warning", button_type="warning")
button3_disabled = Button(label="warning", button_type="warning", disabled=True)
checkbox_grp3 = CheckboxButtonGroup(labels=checkbox_options, active=[0], button_type="warning")
checkbox_grp3_disabled = CheckboxButtonGroup(labels=checkbox_options, active=[0], button_type="warning", disabled=True)
radio_button_group3 = RadioButtonGroup(labels=radio_options, active=0, button_type="warning")
radio_button_group3_disabled = RadioButtonGroup(labels=radio_options, active=0, button_type="warning", disabled=True)
toggle3 = Toggle(label="warning", button_type="warning")
toggle3_disabled = Toggle(label="warning", button_type="warning", disabled=True)

button4 = Button(label="danger", button_type="danger")
button4_disabled = Button(label="danger", button_type="danger", disabled=True)
checkbox_grp4 = CheckboxButtonGroup(labels=checkbox_options, active=[0], button_type="danger")
checkbox_grp4_disabled = CheckboxButtonGroup(labels=checkbox_options, active=[0], button_type="danger", disabled=True)
radio_button_group4 = RadioButtonGroup(labels=radio_options, active=0, button_type="danger")
radio_button_group4_disabled = RadioButtonGroup(labels=radio_options, active=0, button_type="danger", disabled=True)
toggle4 = Toggle(label="danger", button_type="danger")
toggle4_disabled = Toggle(label="danger", button_type="danger", disabled=True)

button5 = Button(label="light", button_type="light")
button5_disabled = Button(label="light", button_type="light", disabled=True)
checkbox_grp5 = CheckboxButtonGroup(labels=checkbox_options, active=[0], button_type="light")
checkbox_grp5_disabled = CheckboxButtonGroup(labels=checkbox_options, active=[0], button_type="light", disabled=True)
radio_button_group5 = RadioButtonGroup(labels=radio_options, active=0, button_type="light")
radio_button_group5_disabled = RadioButtonGroup(labels=radio_options, active=0, button_type="light", disabled=True)
toggle5 = Toggle(label="light", button_type="light")
toggle5_disabled = Toggle(label="light", button_type="light", disabled=True)

from bokeh.models import Slider

slider = Slider(start=0, end=10, value=1, step=0.1, title="Stuff")
slider_disabled = Slider(start=0, end=10, value=1, step=0.1, title="Stuff", disabled=True)

from datetime import date

from bokeh.models import DateRangeSlider

date_range_slider = DateRangeSlider(
    value=(date(2016, 1, 1), date(2016, 12, 31)),
    start=date(2015, 1, 1),
    end=date(2017, 12, 31),
)


date_range_slider_disabled = DateRangeSlider(
    value=(date(2016, 1, 1), date(2016, 12, 31)),
    start=date(2015, 1, 1),
    end=date(2017, 12, 31),
    disabled=True,
)

from bokeh.models import TabPanel, Tabs, TextInput

tab1 = TabPanel(child=TextInput(value="My input", title="TextInput"), title="Input1")
tab2 = TabPanel(child=TextInput(value="My input", title="TextInput"), title="Input2")
tab3 = TabPanel(child=TextInput(value="My input", title="TextInput"), title="Input3")
tab4 = TabPanel(child=TextInput(value="My input", title="TextInput"), title="Input4")
tab5 = TabPanel(
    child=TextInput(value="My input", title="TextInput"),
    title="Multichoice",
    disabled=True,
)
tabs = Tabs(tabs=[tab1, tab2, tab3, tab4, tab5], tabs_location="above")

from bokeh.models import Button, CustomJS, Progress

progress = Progress(
    value=0,
    min=0,
    max=179,
    label="@{index} of @{total} (@{percent}%)",
    description="Description item @{index} of @{total} (@{percent}%)",
    height=300,
    orientation="vertical",
)
progress_h = Progress(
    value=0,
    min=0,
    max=179,
    label="@{index} of @{total} (@{percent}%)",
    description="Description item @{index} of @{total} (@{percent}%)",
    width=300,
)
progress_i = Progress(
    value=4,
    min=0,
    max=5,
    label="@{index} of @{total} (@{percent}%)",
    description="Description item @{index} of @{total} (@{percent}%)",
    height=300,
    orientation="vertical",
)
progress_i_h = Progress(
    value=2,
    min=0,
    max=5,
    label="@{index} of @{total} (@{percent}%)",
    description="Description item @{index} of @{total} (@{percent}%)",
    width=300,
)


button_compu = Button(label="Start computation")
button_compu.js_on_click(
    CustomJS(
        args=dict(progress=progress, button=button_compu, progress_h=progress_h),
        code="""
function delay(ms) {
    return new Promise((resolve) => setTimeout(resolve, ms))
}
export default async ({progress, progress_h, button}) => {
    button.disabled = true
    progress.value = 0
    try {
        while (!progress.has_finished) {
            const ms = Math.random()*100
            await delay(ms)
            progress.increment(1)
            progress_h.increment(1)
        }
    } finally {
        button.disabled = false
    }
}
""",
    ),
)

from bokeh.models import AutocompleteInput
from bokeh.sampledata.world_cities import data

completion_list = data["name"].tolist()

auto_complete_input = AutocompleteInput(title="Enter a city:", completions=completion_list, search_strategy="includes")


from bokeh.models import CheckboxGroup

LABELS = ["Option 1", "Option 2", "Option 3"]

checkbox_group = CheckboxGroup(labels=LABELS, active=[0, 1])
checkbox_group.js_on_change(
    "active",
    CustomJS(
        code="""
    console.log('checkbox_group: active=' + this.active, this.toString())
""",
    ),
)

from bokeh.models import RadioGroup

radio_group = RadioGroup(labels=LABELS, active=0)
radio_group.js_on_event(
    "button_click",
    CustomJS(
        code="""
    console.log('radio_group: active=' + this.origin.active, this.toString())
""",
    ),
)

from bokeh.models import FileInput

file_input = FileInput()

from bokeh.models import NumericInput

numeric_input = NumericInput(value=1, low=1, high=10, title="Enter a number between 1 and 10:")

from bokeh.models import PasswordInput

password_input = PasswordInput(placeholder="enter password...")
password_input.js_on_change(
    "value",
    CustomJS(
        code="""
    console.log('password_input: value=' + this.value, this.toString())
""",
    ),
)

from bokeh.models import Select

select = Select(title="Option:", value="foo", options=["foo", "bar", "baz", "quux"])
select.js_on_change(
    "value",
    CustomJS(
        code="""
    console.log('select: value=' + this.value, this.toString())
""",
    ),
)

from bokeh.models import Spinner

spinner = Spinner(title="Glyph size", low=1, high=40, step=0.5, value=4, width=80)

from bokeh.models import Switch

switch = Switch(active=True, label="Label:")
switch.js_on_change(
    "active",
    CustomJS(
        code="""
    console.log('switch: active=' + this.active, this.toString())
""",
    ),
)

from bokeh.models import TextAreaInput

text_area_input = TextAreaInput(value="default", rows=6, title="Label:")
text_area_input.js_on_change(
    "value",
    CustomJS(
        code="""
    console.log('text_area_input: value=' + this.value, this.toString())
""",
    ),
)


from bokeh.plotting import curdoc

curdoc().theme = "dark_minimal"

# Display
show(
    column(
        children=[
            text_area_input,
            switch,
            spinner,
            select,
            password_input,
            numeric_input,
            file_input,
            auto_complete_input,
            checkbox_group,
            radio_group,
            button_compu,
            row(progress, progress_i),
            progress_h,
            progress_i_h,
            p,
            row(tabs, margin=2),
            date_range_slider,
            date_range_slider_disabled,
            slider,
            slider_disabled,
            row(dropdown, dropdown_disabled),
            row(dropdown1, dropdown1_disabled),
            row(dropdown2, dropdown2_disabled),
            row(dropdown_split, dropdown_disabled),
            row(button, button_disabled),
            row(button1, button1_disabled),
            row(button2, button2_disabled),
            row(button3, button3_disabled),
            row(button4, button4_disabled),
            row(button5, button5_disabled),
            row(checkbox_grp, checkbox_grp_disabled),
            row(checkbox_grp1, checkbox_grp1_disabled),
            row(checkbox_grp2, checkbox_grp2_disabled),
            row(checkbox_grp3, checkbox_grp3_disabled),
            row(checkbox_grp4, checkbox_grp4_disabled),
            row(checkbox_grp5, checkbox_grp5_disabled),
            row(radio_button_group, radio_button_group_disabled),
            row(radio_button_group1, radio_button_group1_disabled),
            row(radio_button_group2, radio_button_group2_disabled),
            row(radio_button_group3, radio_button_group3_disabled),
            row(radio_button_group4, radio_button_group4_disabled),
            row(toggle, toggle_disabled),
            row(toggle1, toggle1_disabled),
            row(toggle2, toggle2_disabled),
            row(toggle3, toggle3_disabled),
            row(toggle4, toggle4_disabled),
            row(toggle5, toggle5_disabled),
            row(help_button, help_button_disabled),
            row(help_button1, help_button1_disabled),
            row(help_button2, help_button2_disabled),
        ],
        sizing_mode="stretch_width",
        stylesheets=[":host { background-color: var(--background-color); color: var(--color);}"],
    ),
)

```

</details>

shows something like (depending on the value of `--color-scheme` in `src/less/base.less`):

|`light`|`dark`|
|-|-|
|<img width="936" height="3326" alt="imagen" src="https://github.com/user-attachments/assets/dbe23a56-831c-460c-88fb-ed0637544150" />|<img width="936" height="3326" alt="imagen" src="https://github.com/user-attachments/assets/d318c02d-cb91-4904-a00d-729eafc2d28d" />|

dalthviz committed:
Update baselines

dalthviz committed:
Merge branch 'branch-3.9' into dalthviz/14445_buttons_tools_design

dalthviz committed:
Update baselines

dalthviz committed:
Merge branch 'branch-3.9' into dalthviz/14445_buttons_tools_design

dalthviz committed:
Update baselines

dalthviz commented:
Closing as superseded by PR https://github.com/bokeh/bokeh/pull/15059 (direct modification of Bokeh styles) and PR https://github.com/bokeh/bokeh/pull/15194 (shows a way to customize widgets styles for the missing styles not directly modified via #15059)
